tests/test_runs/test_run_functions.py

```diff
@@ -97,8 +99,14 @@ def _compare_predictions(self, predictions, predictions_prime):
             # that does not necessarily hold.
             # But with the current code base, it holds.
             for col_idx in compare_slice:
-                self.assertEqual(predictions['data'][idx][col_idx],
-                                 predictions_prime['data'][idx][col_idx])
+                val_1 = predictions['data'][idx][col_idx]
+                val_2 = predictions_prime['data'][idx][col_idx]
+                if type(val_1) == type(val_2):
+                    self.assertEqual(val_1, val_2)
+                elif type(val_1) == float or type(val_2) == float:
+                    self.assertTrue(abs(float(val_1) - float(val_2)) < 0.00001)
```

Comment:
Please replace this by `self.assertAlmostEqual`.

Reply:
Cool. done.